src/Outcome/include/Outcome/Outcome.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
can we include this in OrbitBase/Result.h?

Reply:
If we want that, yes. I wasn't sure if we want to mix that.

Reply:
it is effectively already very much tied to outcome. we use `outcome::success()` everywhere.

Reply:
Okay, so I will move it there. Thanks, Dimitry!

Reply:
Done.